satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -63,13 +65,22 @@ func rootMutation(service *console.Service, mailService *mailservice.Service, ty
 					InputArg: &graphql.ArgumentConfig{
 						Type: graphql.NewNonNull(types.UserInput()),
 					},
+					RegTokenSecret: &graphql.ArgumentConfig{
+						Type: graphql.NewNonNull(graphql.String),
+					},
 				},
 				// creates user and company from input params and returns userID if succeed
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
 					input, _ := p.Args[InputArg].(map[string]interface{})
+					regTokenSecretInput, _ := p.Args[RegTokenSecret].(string)
 					createUser := fromMapCreateUser(input)
 
-					user, err := service.CreateUser(p.Context, createUser)
+					regToken, err := uuid.Parse(regTokenSecretInput)
```

Comment:
This seems problematic. `uuid`-s aren't random and quite predictable, it should be a `crypto/rand` nonce.